digits/standard-networks/tensorflow/alexnet_slim.py

```diff
@@ -0,0 +1,38 @@
+from model import Tower
```

Comment:
They are two different approaches. One could even make a model with keras. It's up to the user. I don't mind much. What do you think @lukeyeager ? We can remove the slim versions?